Playbooks/NewIncidentTriggerPlaybook/readme.md

```diff
@@ -0,0 +1,19 @@
+# Alert trigger empty playbook
+Use this template to quickly create a new playbook which starts with an Azure Sentinel alert.<br>
```

Comment:
this is in the incident folder but says alert

Reply:
Thanks! reordered folders to correct this + have a similar structure.

These playbooks will allow a quick start from gallery